var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,20 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=False, description='Enable MPI support')
```

Comment:
This should probably default to True. It is a _supercomputing_ package manager after all. See #2590 for more reasons why we want all MPI variants to default to True.